cargo-dist/templates/ci/github/release.yml.j2

```diff
@@ -577,6 +577,7 @@ jobs:
           owner: {{{ github_releases_repo.owner }}}
           repo: {{{ github_releases_repo.repo }}}
           token: ${{ secrets.GH_RELEASES_TOKEN }}
+          commit: ${{ fromJson(needs.host.outputs.val).ci.github.external_repo_commit }}
```

Comment:
This needs to be wrapped in some kind of if, since it should only be done if the submodule config is set

Reply:
I believe if it's passed as `null`, it's functionally the same as if it weren't passed at all - that's why I didn't do any kind of "if" here. https://github.com/ncipollo/release-action/blob/1cbdc80532ba923d3d81ebd1cece305c437b8d66/src/Inputs.ts#L110-L116

Reply:
Will that still work right if we make `external_repo_commit` skip_serializing_if?

Reply:
(probably?)

Reply:
After we did a little looking, we've settled on that this should work as-is.